language-design.md

```diff
@@ -17,7 +17,82 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+1. From the article by Pavlus:
+
+>"According to findings by researchers from Southern Illinois University, this
+>reaction isn’t just because you’re a n00b: they found that Perl, a major
+>programming language used by untold zillions of developers, is no more
+>intuitive to novices than a language with a randomly generated syntax."
+>[Pavlus, 2012].
+
+We selected this quote as an example of something we disagree with.  Most
+notably, the quote contradicts itself.  The quote first says that if you
+have trouble intuiting code then it's not because you're a n00b.  However,
+they go on to reference a studying that tested people who have never coded
+before.  This quote does not support their point that experienced coders
+struggle with syntax.
+
+We also disagree with the sentiment that programmers should employ syntax more
+intuitive to novices.  Sure, using natural language could look more
+understandable for someone who has never coded before.  However, for more
+experienced coders, using natural language could feel ambiguous and cumbersome.
+Frequent users of code would appreciate a concise set of keywords that they
+could build off of.  Also, we think it is more important to cater to experienced
+coders than the "n00bs" in the study because they were only novices for an hour.
+Typically, people spend more time as experienced coders who understand the
+syntax than as beginners trying to read off code like a natural language.  There
```

Comment:
Your comment on "trying to read off code like a natural language" made me think about how I read code. A lot of people sound out each word in their head when they read natural language. I do a similar thing in math, with some symbol heavy exceptions such as when reading {a_1, ..., a_n}, which I read as "a set containing a one through a n."

For code, I don't read off every symbol when I see a line such as
```
for (int i = 0; i < 10; ++i) {
}
```
but I do read it in "natural language". I think I would read this loop as "a for loop that runs 10 times." But I may be overthinking this concept, I'm not sure if I actually sound it out like that in my head. But I am sure that I don't read every symbol, because when confronted with a non-idiomatic loop such as
```
for (int i = 0; i <= 10; ++i) {
}
```
I slow down and pay more attention to what the for loop is actually stating. I'm curious to know what other people "hear" in their head when they read a line of code.